test/runtests.jl

```diff
@@ -52,33 +56,47 @@ using Test
                 :LUFactorization,
                 :QRFactorization,
                 :SVDFactorization,
-                :DefaultFactorization,
-                :DefaultLinSolve
+    #           :DefaultLinSolve
                )
         test_interface(alg, kwargs, prob1, prob2, prob3)
     end
 
+#   alg = :DefaultFactorization
+#   for fact_alg in (
+#                    :lu, :lu!,
+#                    :qr, :qr!,
+#                    :cholesky, :cholesky!,
+#   #                :ldlt, :ldlt!,
+#                    :bunchkaufman, :bunchkaufman!,
+#                    :lq, :lq!,
+#                    :svd, :svd!,
+#                    :(LinearAlgebra.factorize), 
+#                   )
+#       kwargs = :(fact_alg=$fact_alg,)
+#       test_interface(alg, kwargs, prob1, prob2, prob3)
+#   end
```

Comment:
Never comment out tests 😱

Reply:
Do these just not apply anymore? If so, remove them if the same pieces are tested elsewhere.

Reply:
i wrote them messily. was commenting to find erroring alg. using testsets instead